iron-location.html

```diff
@@ -261,18 +261,32 @@
         return null;
       }
       var eventPath = Polymer.dom(event).path;
-      var href = null;
+      var anchor = null;
       for (var i = 0; i < eventPath.length; i++) {
         var element = eventPath[i];
         if (element.tagName === 'A' && element.href) {
-          href = element.href;
+          anchor = element;
           break;
         }
       }
+
       // If there's no link there's nothing to do.
-      if (!href) {
-        return null;
+      if (!anchor) {
+        return;
+      }
+
+      // Target blank is a new tab, don't intercept.
+      if (anchor.target === '_blank') {
+        return;
       }
+      // If the link is for an existing parent frame, don't intercept.
+      if ((anchor.target === '_top' ||
+           anchor.target === '_parent') &&
+          window.top !== window) {
+        return;
+      }
```

Comment:
🚲 🏠: I tend to prefer a mega-conditional (as ugly as that can be to read) in favor of multiple short-circuiting return statements in cases like this. We don't have a formal style guide and this is a personal preference, so please consider this a bike shed with no required action.